run.sh

```diff
@@ -11,44 +12,72 @@ echo "# Ark Server - " `date`
 echo "###########################################################################"
 
 echo "Ensuring correct permissions..."
-sudo find /ark -not -user steam -o -not -group steam -exec chown -v steam:steam {} \; 
+sudo find /ark -not -user steam -o -not -group steam -exec chown -v steam:steam {} \;
 sudo find /home/steam -not -user steam -o -not -group steam -exec chown -v steam:steam {} \;
 
+if [ -n "$ARKSERVER_SHARED" ]; then
+  # directory is created when something is mounted to 'Saved'
+  [ -d "$ARKSERVER_SHARED/ShooterGame" ] && sudo chown steam:steam $ARKSERVER_SHARED/ShooterGame
+  echo "Shared server files in $ARKSERVER_SHARED..."
+  if [ -z "$(mount | grep "on $ARKSERVER_SHARED/ShooterGame/Saved ")" ]; then
+    echo "===> ABORT !"
+    echo "You seem to be using a shared server directory: '$ARKSERVER_SHARED'"
+    echo "But you have NOT mounted your game instance saved directory to '$ARKSERVER_SHARED/ShooterGame/Saved'"
+    exit 1
+  fi
+  # Shared server files does not support staging directory
+  export am_arkStagingDir=
+fi
+
+if [ "$ARKCLUSTER" = "true" ]; then
+  # directory is created when something is mounted to 'clusters'
+  [ -d "$ARKSERVER_SHARED/ShooterGame/Saved" ] && chown steam:steam $ARKSERVER_SHARED/ShooterGame/Saved
```

Comment:
```suggestion
  [ -d "$ARKSERVER/ShooterGame/Saved" ] && chown steam:steam $ARKSERVER/ShooterGame/Saved
```
Should it be ARKSERVER instead of ARKSERVER_SHARED, for people not using ARKSERVER_SHARED?
Same comment for the few following lines

Reply:
So far, `ARKCLUSTER` required using `ARKSERVER_SHARED` - have you tried it without?

If you're change makes it work, then I'll apply the suggestion, @edausq

Reply:
did you have a chance to test it @edausq?

Reply:
I have tested without ARKCLUSTER nor ARKSERVER_SHARED, and I could make my cluster work without issue, only with a volume.

Reply:
Ok, I'll apply your changes and also use a different variable in the following line.